setup.cfg

```diff
@@ -0,0 +1,3 @@
+[tool:pytest]
+addopts = -nauto
+norecursedirs = vendor
```

Comment:
Need to exclude this from copier files, or move it to pyproject.toml if pytest supports that.